index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
Thanks for the reply. We seem to agree on many points.
I did not understand that your main concern was the unbalanced visual design. I am convinced this can be solved with design without hurting the user journey ("I have 3 seconds for the site, I want to understand what it is")

I would like to keep the tagline too and would prefer not remove it from the page.

I like your latest iteration. 
It satisfies my main criteria to have a hero message conveying what Bazel is and its value.
it is also visually better than the current state.